apps/root/src/hooks/useStoredContactList.ts

```diff
@@ -19,7 +21,9 @@ function useStoredContactList() {
 
   const labeledContactList = React.useMemo<ContactList>(() => {
     const contactsWithWallets = uniq([...map(storedContactList, 'address'), ...map(wallets, 'address')]);
-    return contactsWithWallets.map((address) => ({ address, label: labels[address] }));
+    return contactsWithWallets
+      .filter((address) => address !== activeWallet?.address)
```

Comment:
Mmm I think it would be better to filter this on the contacts modal directly so we can use storedContactList everywhere to check if the address is already a contact regardless of the active wallet